packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue

```diff
@@ -36,9 +36,14 @@
 				<template #actions>
 					<slot name="header-actions" />
 					<tera-operator-annotation
+						v-if="activeOutputSummary === undefined"
 						:state="node.state"
 						@update-state="(state: any) => emit('update-state', state)"
 					/>
+					<!-- TODO: Make this as a summary component -->
+					<div v-else>
+						{{ activeOutputSummary === '' ? 'Generating AI summary...' : activeOutputSummary }}
+					</div>
```

Comment:
These are two separate concept, they are not, I don't think they are meant to replace one another
- State.annotation: The notes made on the operator
- WorkflowOutput.summary: The summary/note of the output

So you can have one State.annotation but many Worlfowoutput.summary.

I think we should revert the changes here for now, and get clarifications from Pacale and the design team on what they want.